linux_os/guide/services/obsolete/r_services/no_host_based_files/ansible/shared.yml

```diff
@@ -5,8 +5,7 @@
 # disruption = low
 - block:
     - name: "Find local mount points"
-      shell: |
-          set -o pipefail
+      shell: | #noqa 306 we don't care about grep failure in this case
```

Comment:
Unfortunately the yaml libraries strip out comments when processing them.
The exception for ansible-lint should be added here: https://github.com/ComplianceAsCode/content/blob/master/tests/ansible-lint_config.yml